mrglog.py

```diff
@@ -905,6 +905,11 @@ def addTestId(self, testid=None, result='FAIL'):
                     'tag': 'testid', 'attrs': {'result': str(result).upper()}})
 
     @property
+    def pass_(self):
```

Comment:
For others there's plural used so even for pass there should be plural and it solve the conflict more easily than to use '\_'. I.e. 'passes' instead of 'pass\_'.